package.json

```diff
@@ -30,6 +30,7 @@
     "colors": "0.6.0-1",
     "connect-phonegap": "0.21.2",
     "cordova": "6.1.1",
+    "insight": "^0.8.2",
```

Comment:
We try to lock dependencies to avoid unexpected surprised. Please remove the `^`.